site/plugins/sidebar/sidebar.py

```diff
@@ -172,21 +167,16 @@ def _prepare_sidebar(self, destination, lang, template):
             context['global_{}_hierarchy'.format(taxonomy)] = taxonomy_hierarchy
             deps_dict['global_{}_hierarchy'.format(taxonomy)] = taxonomy_hierarchy
 
-        archive_list = self._build_month_post_list(lang)
-        context['archive_list'] = archive_list
-        deps_dict['archive_list'] = []
+        month_list = self._build_month_post_list(lang)
+        context['month_list'] = month_list
+        deps_dict['month_list'] = []
+
 
-        for year, months in archive_list:
-            deps_dict['archive_list'] += [(month.year + '/' + month.month, month.friendly_name) for month in months]
+        deps_dict['month_list'] += [(month.year + '/' + month.month, month.friendly_name) for month in month_list]
```

Comment:
no entiendo bien por que le agregás una lista vacia y acto seguido le sumás otra lista...

Reply:
Porque quedó feíto ... feíto feíto jajaja